src/error/specification

```diff
@@ -1209,3 +1209,39 @@ severity:error
 ingroup:plugin
 module:typechecker
 macro:PLUGIN_TYPECHECKER_SET
+
+number:194
+description:The given path does not meet the correct permissions
+severity:error
+ingroup:plugin
+module:path
+
+number:195
+description:Could not find the given user in the system
+severity:error
+ingroup:plugin
+module:path
+
+number:196
+description:The given port is not in a valid range (0-65535)
+severity:error
+ingroup:plugin
+module:network
+
+number:196
+description:The desired port is already in use.
```

Comment:
Please be consistent with the already existing errors if you put a . in the end or not.

Reply:
Some descriptions have a dot and some do not. Which one should i take now?

Reply:
Ok, this is something to unify later.

We can also make that sentences get a .